proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -200,3 +203,18 @@ message EdgeDecisionServiceConfig {
     // specify regexes which will be matched with http.url attribute to exclude spans from EdgeDecisionService evaluation.
   repeated google.protobuf.StringValue exclude_path_regexes = 5;
 }
+
+message MetricsExporterConfig {
+  // when `true` metrics will be exported as spans from libtraceable
+  google.protobuf.BoolValue enabled = 1;
+  // use HTTPS for calls to TPA
+  google.protobuf.BoolValue secure = 2;
+  // endpoint for reporting the spans
+  google.protobuf.StringValue endpoint = 3;
```

Comment:
Why is this config needed?

Reply:
Wanted to configure the metrics exporter pipeline of libtraceable and we can have metrics for edge decision service based deployments
Pr in libtraceable for edge decision service metrics: Pls review when you get some time. Pr for edge decision service metrics: https://github.com/Traceableai/libtraceable/pull/390

Reply:
We never turned this feature on from libtraceable, so no config as of now

Reply:
ok.